e2e/constants.js

```diff
@@ -11,8 +11,9 @@ export const MISSION_TIME = 1732413600000; // Saturday, November 23, 2024 6:00:0
 
 /**
  * URL Constants
- * - This is the URL that the browser will be directed to when running visual tests. This URL 
+ * - This is the URL that the browser will be directed to when running visual tests. This URL
  *    - hides the tree and inspector to prevent visual noise
  *    - sets the time bounds to a fixed range
  */
-export const VISUAL_URL = './#/browse/mine?tc.mode=fixed&tc.startBound=1693592063607&tc.endBound=1693593893607&tc.timeSystem=utc&view=grid&hideInspector=true&hideTree=true';
+export const VISUAL_FIXED_URL =
```

Comment:
ah, got it. Took me too long reading the actual value to realize it was the name which changed

Reply:
i think i'm gonna change it back